l10n_nl_account_tax_unece/post_install.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2017 Akretion (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2017-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, SUPERUSER_ID
+
+
+def set_unece_on_taxes(cr, _):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        for company in env['res.company'].search([]):
+            if company.country_id and company.country_id != env.ref('base.nl'):
```

Comment:
Would it not be more safe to have this only work for companies that have country_id explicitly set to the Netherlands?

Reply:
Right, thanks!